Src/zipkin4net/Src/Tracers/Zipkin/JSONSpanSerializer.cs

```diff
@@ -126,7 +126,7 @@ private static void SerializeEndPoint(StreamWriter writer, IPEndPoint endPoint,
             writer.Write(comma);
             writer.WriteField(port, (short)endPoint.Port);
             writer.Write(comma);
-            writer.WriteField(JSONSpanSerializer.serviceName, serviceName);
+            writer.WriteField(JSONSpanSerializer.serviceName, SerializerUtils.ToEscaped(serviceName));
```

Comment:
Shouldn't we also escape annotation.value on line 115 ?

Reply:
I find the same problem in serviceName too, so I add it too
I thought annotation.Value is fully controlled by us, so I didn't add it
I commit again